packages/babel-cli/src/babel/index.js

```diff
@@ -45,6 +45,7 @@ commander.option("-o, --out-file [out]", "Compile all input files into a single
 commander.option("-d, --out-dir [out]", "Compile an input directory of modules into an output directory");
 commander.option("-D, --copy-files", "When compiling a directory copy over non-compilable files");
 commander.option("-q, --quiet", "Don't log anything");
+commander.option("--copy-dot-files", "When compiling a directory copy over dot files", false);
```

Comment:
Can we change the message to say something like
`Enables copying of dot files when using --copy-files.`

At least it would be better to understand if we mention that you need to do --copy-files